document/custom.css

```diff
@@ -0,0 +1,19 @@
+a {
+  text-decoration: none;
+}
+
+a.reference {
+	border-bottom: none;
```

Comment:
Indentation consistency (2 spaces, 1 tab or four spaces), but not all three in one file :P